Classes/Command/TradosCommandController.php

```diff
@@ -40,25 +41,33 @@ class TradosCommandController extends CommandController
      *
      * This command exports a specific site including all content into an XML format.
      *
-     * @param string $startingPoint The node with which to start the export, relative to the site node. Optional.
+     * @param string $startingPoint The node with which to start the export: as identifier or the path relative to the site node.
      * @param string $sourceLanguage The language to use as base for the export.
-     * @param string $targetLanguage The target language for the translation, optional.
-     * @param string $filename Path and filename to the XML file to create.
-     * @param string $modifiedAfter
+     * @param string|null $targetLanguage The target language for the translation, optional.
+     * @param string|null $filename Path and filename to the XML file to create.
+     * @param string|null $modifiedAfter
      * @param boolean $ignoreHidden
      * @return void
+     * @throws \Exception
      */
-    public function exportCommand($startingPoint, $sourceLanguage, $targetLanguage = null, $filename = null, $modifiedAfter = null, $ignoreHidden = true)
+    public function exportCommand(string $startingPoint,
+                                  string $sourceLanguage,
```

Comment:
No indented & aligned arguments, please.

Reply:
is this a Neos coding guideline or your personal preference? 
because when there are many arguments with type hinting and default values, these lines become very long (almost 200 characters) and hard to read.
and in PSR-2 it is allowed to split arguments across multiple lines, I would just have to fix the indentation to match the standard.
https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#44-method-arguments